performance_manager/lib/l1_full_trip_events.py

```diff
@@ -56,7 +57,13 @@ def pull_and_transform(db_manager: DatabaseManager) -> pandas.DataFrame:
         sa.select(
             VehiclePositionEvents.pk_id.label("fk_vp_moving_event"),
             VehiclePositionEvents.stop_sequence,
-            VehiclePositionEvents.stop_id,
+            sa.case(
+                (
+                    StaticStops.parent_station.is_(None),
+                    VehiclePositionEvents.stop_id,
+                ),
+                else_=StaticStops.parent_station,
+            ).label("parent_station"),
```

Comment:
replace `stop_id` select with case statement

not all stop id's have a parent station, so if not parent station is found, use stop_id